components/brave_rewards/resources/page/components/connect_wallet_modal.tsx

```diff
@@ -0,0 +1,247 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import { LocaleContext, formatMessage } from '../../shared/lib/locale_context'
+import { Modal, ModalCloseButton } from '../../shared/components/modal'
+import { NewTabLink } from '../../shared/components/new_tab_link'
+import { CaretIcon } from '../../shared/components/icons/caret_icon'
+import { GeminiIcon } from '../../shared/components/icons/gemini_icon'
+import { UpholdIcon } from '../../shared/components/icons/uphold_icon'
+import { BitflyerIcon } from '../../shared/components/icons/bitflyer_icon'
+import { termsOfServiceURL } from '../../shared/components/terms_of_service'
+
+import connectWalletGraphic from '../assets/connect_wallet.svg'
+
+import * as style from './connect_wallet_modal.style'
+
+function Check () {
```

Comment:
Nit: Maybe call it `CheckIcon` just to be super clear?

Reply:
Good call